Apps/Playground/Scripts/config.json

```diff
@@ -270,22 +270,22 @@
         },
         {
             "title": "Scissor test",
-            "playgroundId": "#W7E7CF#34",
+            "playgroundId": "#W7E7CF#36",
             "referenceImage": "scissor-test.png",
             "excludedGraphicsApis": [ "D3D12" ],
             "comment": "TODO: reenable D3D12 when automatic mip-maps issue is fixed in bgfx"
         },
         {
             "title": "Scissor test with 0.9 hardware scaling",
-            "playgroundId": "#W7E7CF#34",
+            "playgroundId": "#W7E7CF#36",
             "replace": "//options//, hardwareScalingLevel = 0.9;",
             "referenceImage": "scissor-test-2.png",
             "excludedGraphicsApis": [ "D3D12" ],
             "comment": "TODO: reenable D3D12 when automatic mip-maps issue is fixed in bgfx"
         },
         {
             "title": "Scissor test with 1.5 hardware scaling",
-            "playgroundId": "#W7E7CF#34",
+            "playgroundId": "#W7E7CF#36",
```

Comment:
We have the same tests for Babylon.js. Why is it not an issue there?

Reply:
Fix is not correct. Root cause in PR description.